src/main/resources/static/js/minimal.lightbox.js

```diff
@@ -0,0 +1,56 @@
+var imagesArray = document.getElementsByTagName('img');
+
+function minimalLightbox() {
+    var imageSelect = this.src;
+    if(this.getAttribute('minimal-lightbox') != '')
+        imageSelect = this.getAttribute('minimal-lightbox');
+    var topLayer = document.createElement('div');
+    var displayImage = document.createElement('div');
+    var closeAll = document.createElement('a');
+
+    topLayer.setAttribute('id', 'top-layer');
+    document.body.appendChild(topLayer);
+
+    displayImage.setAttribute('id', 'display-image');
+    document.body.appendChild(displayImage);
+
+    closeAll.setAttribute('id', 'close');
+    displayImage.appendChild(closeAll);
+    closeAll.innerHTML = '[X]';
+
+    displayImage.addEventListener('click', closeImage);
+    window.addEventListener('keydown', closeImage);
+    closeAll.addEventListener('click', closeImage);
+
+    topLayer.setAttribute('style', 'height: 100%; width: 100%; top: 0; left: 0; position: fixed; background-color: rgba(0, 0, 0, 0.9);');
+    displayImage.setAttribute('style','position: fixed; left: 20%; top: 5%; width: 70%; height: 70%; background-image: url("' + imageSelect + '");  background-size: contain; background-position: center center; background-repeat: no-repeat;');
+    closeAll.setAttribute('style', 'position: fixed; right: 20px; top: 20px; font-size: 20px; color: #FFF; cursor: pointer;');
+    body = document.getElementsByTagName("body")[0];
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add the "let", "const" or "var" keyword to this declaration of "body" to make it explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS2703)